rllib/algorithms/algorithm.py

```diff
@@ -670,6 +671,12 @@ def setup(self, config: AlgorithmConfig) -> None:
                     "offline.offline_evaluator::OfflineEvaluator"
                 )
 
+        self.trainer_runner = None
+        if self.config._enable_rl_trainer_api:
+            policy = self.get_policy()
+            trainer_runner_config = self.config.get_trainer_runner_config(policy)
```

Comment:
I suggest you only pass in the actual useful bits, like obs and action spaces, so folks don't feel the freedom to use arbitrary data.
policy is also an old world concept, requiring policy for trainer_runner to work looks weird.